app.config.js

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2018 - 2022 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+const IS_DEV = process.env.APP_VARIANT === 'development'
+
+export default {
+  expo: {
+    name: IS_DEV ? 'Alephium Wallet (DEV)' : 'Alephium Wallet',
```

Comment:
Interesting. Is this a recommended approach? (same remark regarding the different bundleIdentifiers)

I've never seen this before, usually it's the same app name, but different release pipelines (for alpha, beta etc.).

Reply:
I saw this in the expo docs. 

https://docs.expo.dev/build-reference/variants/

It doesn't affect alpha, beta, etc. It's for us to be able to have 2 apps on our devices, both prod (for personal use) and dev (for, well, development).